### ISSUE TITLE
Unable to run Cypress tests on gitlab using cypress-cloud

### ISSUE THREAD

**Issue by PavanGurram-DevOps** [sorry-cypress/sorry-cypress#861]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/sorry-cypress/sorry-cypress/issues?q=is%3Aopen+is%3Aissue) and [discussions](https://github.com/sorry-cypress/sorry-cypress/discussions).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have read the [documentation](https://docs.sorry-cypress.dev/) and found no answer to my problem

### Environment information

<details>

```
  - cypress-cloud version: `1.9.4`
  - cypress version: `13.3.0`
  - platform: `k8s | docker | aws | gitlabci`
  - service: `all`

```

</details>

### Describe the bug

We are trying to implement sorry-cypress into our environments and unable to execute tests on gitlab ci provider. I have tried many trail and errors but no luck. The log itself doesn't show specs are found etc. I have already gone through few issues closed in gitlab repo but no luck too. I have enabled debug options and below is the full log. It doesn't look like an environment setup but you can look and confirm it please.

I have made sure currents and cypress config files are at root level with correct configuration.

### Expected behavior

Cypress tests should get execute.

### Reproduction steps

Please let me know what other details are needed to debug the issue

### Full log and debug output

Please find the full log attached, thanks

**Comment by DamienCassou** [comment]
Cypress v13 is not supported: https://currents.dev/posts/v13-blocking

**Comment by samixchoumi** [comment]
When downgrading to latest maintained version, don't forget to clear cache or you will still have the message for some jobs.

**Comment by PavanGurram-DevOps** [comment]
Thanks @DamienCassou  @samixchoumi, it is resolved by downgrading cypress version and deleting cache. One last question before closing this issue - how to pass tags in the cli command? I have tried to search in documentation but no luck. I used below command to run in gitlab ci, have tried with few more options though but no luck

npx cypress-cloud --parallel --record --key ui-admiralmoney-public-key --ci-build-id "$CI_PIPELINE_ID" --env='TAGS=@regression' --env='ENV=dev'

**Comment by DamienCassou** [comment]
I don't know the answer to your question and it seems unrelated to the issue. I suggest closing the issue and asking for help in the [Slack channel](https://docs.sorry-cypress.dev/support) or opening a new issue

**Comment by PavanGurram-DevOps** [comment]
Closing it as suggested, thanks

**Comment by samixchoumi** [comment]
@PavanGurram-DevOps Just to answer you about the TAGS question, do something like that : 

On your package.json file add it like that `--env TAGS=\"@testenv\"`. It should work 👍

**Comment by PavanGurram-DevOps** [comment]
@samixchoumi Thanks for the response. I have tried as below but no luck. They work with cypress binary but not with cypress-cloud. Please help if you can, thanks

 "cy:cloud": "npx cypress-cloud --parallel --record --key ui-admiralmoney-public-key --env TAGS=\"@regression\" --env ENV=\"dev\"",

**Comment by samixchoumi** [comment]
@PavanGurram-DevOps actually my script looks like that : 

`"test": "npx cypress-cloud run --record --key aaa --parallel --ci-build-id $CI_PIPELINE_ID --env TAGS=\"not @ignore\" --spec '**/!(TestsTeardown).feature' --browser chrome",`

Maybe you have something wrong on your feature file.

**Comment by PavanGurram-DevOps** [comment]
@samixchoumi  It worked now. Problem is not with feature file, I'm passing 2 env vars with --env 2 times. If I use --env only once and pass more than 1 value with comma separated then it passes. Thanks for your support.

**Comment by rrauenza** [comment]
> Cypress v13 is not supported: https://currents.dev/posts/v13-blocking

FYI: My v12 cypress suddenly started blocking as well.

**Comment by samixchoumi** [comment]
@rrauenza see here : https://github.com/currents-dev/cypress-cloud/issues/193#issuecomment-1800741523
